modules/tinymce/changelog.txt

```diff
@@ -3,6 +3,7 @@ Version 5.2.2 (2020-04-22)
     Fixed text decorations (underline, strikethrough) not consistently inheriting the text color #TINY-4757
     Fixed `format` menu alignment buttons inconsistently applying to images #TINY-4057
     Fixed the floating toolbar drawer height collapsing when the editor is rendered in modal dialogs #TINY-4837
+    Fixed the `media` plugin embed content not being processed safely in some cases #TINY-4857
```

Comment:
Fixed TinyMCE not processing embed content from the `media` plugin safely in some cases.

Maybe?